source/_docs/domains.md

```diff
@@ -195,21 +195,15 @@ Replace `http://` in the site's database and configure your CMS to assume users
 ### CNAME Record Workaround
 If your domain's DNS configuration relies on an existing MX or TXT record that intentionally disallows CNAME records, you'll need to use A and AAAA records to configure DNS for subdomains (e.g., `www.example.com`) instead of CNAMEs:
 
-1. Identify required A record value by querying the target environment's [platform domain](#platform-domains) (e.g., `live-example.pantheonsite.io`) from the command line using dig:
+1. Identify required A record value by querying the target environment's [platform domain](#platform-domains) (e.g., `live-example.pantheonsite.io`) using a free online tool: [https://www.whatsmydns.net/#A/{live-example.pantheonsite.io}](https://www.whatsmydns.net/#A/{live-example.pantheonsite.io}).
```

Comment:
Suggest removing `{` and `}` from the URL to reduce confusion around how to structure the URL. `live-example.pantheonsite.io` should be enough to signal example platform domain.